CatMap/README.md

```diff
@@ -0,0 +1,33 @@
+# Cat Map
+
+I wanted to make a cute cat map.
+You can only reach the bottom whiskers with a balloon, but if you can make it, the cyan district becomes powerful.
+Beware getting stuck on the eyes!
+
+## Screenshots
+
+![cat.frb](cat.png)
+
+## Features
+
+| Map Properties    | Value      |
+| ----------------- | ---------- |
+| Initial Cash      | 2240       |
+| Target Amount     | 32000      | 
+| Base Salary       | 320        | 
+| Salary Increment  | 240        | 
+| Maximum Dice Roll | 8          | 
+| Looping Mode      | No Looping | 
+
+## Prerequisites
+
+- Needs to replace a map that supports cannons.
```

Comment:
```suggestion
- Needs to replace a dragon quest themed map.
```

Reply:
I wouldn't word it exactly like this. I think I remember enjoying the theme most when it replaces Yoshi's Island, despite the graphical glitches on the 2 way doors. Will say 

- Needs to replace a map that supports cannons.
- Should replace a dragon quest themed map to avoid graphical glitches when using a 2 way doors.